net.sourceforge.Artha.yaml

```diff
@@ -0,0 +1,141 @@
+app-id: net.sourceforge.Artha
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: artha
+
+rename-desktop-file: artha.desktop
+#rename-icon: artha.png
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  # dbus session bus for artha
+  - --own-name=org.artha.unique
```

Comment:
this should actually be fixed, it's a violation of the spec to use "random" dbus names.

Reply:
how? I got this working after so much headache. https://discourse.flathub.org/t/dbus-session-permssions-for-artha-dictionary/1000

Reply:
By submitting a patch upstream.

Reply:
`artha.org` belong to someone that doesn't appear to be the developers.

Reply:
well, I don't have coding skills. and what a pleasant surprise, it was you who gave that solution to me :-)

Reply:
that's doesn't remove anything. I was just indicating how to debug it. It doesn't make it right.